File: ObjectAvoidance/java/src/main/java/elite/math/Symbol.java
package elite.math;

/**
 *
 * @author CFerg (Elite)
 */
public class Symbol {
    public enum Symbols{
        CDF ("F(x)"),
        PDF ("f(x)"),
        SUMMATION ("∑");
        
        private final String symbol;
        private
        
        Symbols(String symbol){
            this.symbol = symbol;
        }
        
        public String symbol(){return this.symbol;}
    }
}

File: ObjectAvoidance/src/elite/math/Symbol.kt
package elite.math

/**
 *
 * @author CFerg (Elite)
 */
class Symbol {
    enum class Symbols private constructor(private val symbol: String) {
        CDF("F(x)"),
        PDF("f(x)"),
        SUMMATION("∑");

        fun symbol(): String {
            return this.symbol
        }
    }
}
